auraescript/src/lib.rs

```diff
@@ -27,6 +27,63 @@
  *   limitations under the License.                                           *
  *                                                                            *
 \* -------------------------------------------------------------------------- */
+//! # AuraeScript
+//!
+//! AuraeScript is a turing complete language for platform teams built on [Deno](https://deno.land).
+//!
+//! AuraeScript is a lightweight client that wraps the [Aurae Standard Library](https://aurae.io/stdlib/).
+//!
+//! AuraeScript is a quick way to access the core Aurae APIs and follows normal UNIX parlance. AuraeScript should feel simple and intuitive for any Go, C, Python, or Rust programmer.
+//!
+//! ```typescript
+//! // @ts-ignore
+//! import {AllocateCellRequest, Cell, CellServiceClient} from "../lib/runtime.ts";
+//!
+//! // @ts-ignore
+//! Deno.core.initializeAsyncOps();
+//!
+//! let cells = new CellServiceClient();
+//!
+//! cells.Allocate(<AllocateCellRequest>{
+//!     cell: Cell.fromPartial({
+//!         name: "test",
+//!         cpus: "2"
+//!     })
+//! }).then(r => {
+//!     // @ts-ignore
+//!     Deno.core.print("done")
+//! });
+//! ```
+//!
+//! ## Build From Source
+//!
+//! ⚠️ Early Active Development ⚠️
+//!
+//! ```
+//! git clone git@github.com:aurae-runtime/aurae.git
+//! cd aurae
+//! make pki config all
+//! ```
+//!
+//! Alternatively it is possible to build `auraescript` by itself. Check out this repository and use the Makefile.
+//!
+//! ```bash
+//! make auraescript
+//! ```
+//!
+//! ...or manually using Cargo.
```

Comment:
Remove all references to building with Cargo, we are only supporting `make` at this time